internal/tiltfile/helm.go

```diff
@@ -2,16 +2,24 @@ package tiltfile
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
 	"os/exec"
+	"path/filepath"
 	"regexp"
 	"runtime"
 	"strings"
+
+	"github.com/ghodss/yaml"
```

Comment:
i'm a little bit concerned about how this other yaml library slipped in, as opposed to the one kubernetes uses

Reply:
Fun fact: it looks like kubernetes/kubernetes uses three different YAML libraries: https://github.com/kubernetes/kubernetes/blob/master/go.mod

Reply:
(And to be clear: this is one of them)

Reply:
:open_mouth: